src/main/java/de/teamlapen/vampirism/core/ModVillage.java

```diff
@@ -0,0 +1,229 @@
+package de.teamlapen.vampirism.core;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import it.unimi.dsi.fastutil.ints.Int2ObjectOpenHashMap;
+
+import de.teamlapen.vampirism.items.BloodBottleIItem;
+import de.teamlapen.vampirism.util.REFERENCE;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.ai.brain.schedule.Activity;
+import net.minecraft.entity.ai.brain.schedule.Schedule;
+import net.minecraft.entity.ai.brain.schedule.ScheduleBuilder;
+import net.minecraft.entity.merchant.villager.VillagerProfession;
+import net.minecraft.entity.merchant.villager.VillagerTrades;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.item.MerchantOffer;
+import net.minecraft.util.IItemProvider;
+import net.minecraft.village.PointOfInterestType;
+import net.minecraftforge.registries.IForgeRegistry;
+import net.minecraftforge.registries.ObjectHolder;
+
+import javax.annotation.Nullable;
+import java.util.Random;
+
+import static de.teamlapen.lib.lib.util.UtilLib.getNull;
+
+@ObjectHolder(REFERENCE.MODID)
+public class ModVillage {
+
+    public static final VillagerProfession hunter_expert = getNull();
+    public static final VillagerProfession vampire_expert = getNull();
+
+    public static final PointOfInterestType hunter_faction = getNull();
+    public static final PointOfInterestType vampire_faction = getNull();
+
+    public static final Schedule converted_default = getNull();
+
+    public static final VillagerTrades.ITrade[] converted_trades = new VillagerTrades.ITrade[]{new VillagerTrades.EmeraldForItemsTrade(ModItems.human_heart, 9, 2, 2), new VillagerTrades.ItemsForEmeraldsTrade(ModItems.human_heart, 3, 9, 2), new ModVillage.ItemsForEmeraldsTradeWithDamage(BloodBottleIItem.getStackWithDamage(9), 1, 3, 12, 2)};
```

Comment:
@ObjectHolder classes should not have any public static final fields that are not object holders